### ISSUE TITLE
Buidling code on master results in error when executing any task

### ISSUE THREAD

**Issue by daniel-lohausen-consilica** [aws/aws-toolkit-azure-devops#522]
**Describe the bug**
Building code on master results in error when executing any task.

**To reproduce**
1. Checkout master branch
2. Run `npm install && npm run fullBuild `
3. Run `node ./package/Tasks/AWSCLI/AWSCLI.js` (or any other task)

The following error occurs:
```
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '/mnt/c/Users/UI725000/git/aws-toolkit-azure-devops/build/npmcache/package/Tasks/AWSCLI/AWSCLI.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
```

Same applies when publishing the module and trying in a 'real' environment.
Tested with multiple node/npm versions

**Expected behavior**
- Task is running in real enviorment
- Task will start, but then complain about missing parameters when running locally

**Additional context**
- Works when checking out the [latest release](https://github.com/aws/aws-toolkit-azure-devops/tree/release/v1.13.0)
- Might relate to https://github.com/microsoft/azure-pipelines-task-lib/issues/942

**Comment by daniel-lohausen-consilica** [comment]
@stevejroberts - As the main collaborator, can you help me here? I would like to open a PR for #511 but am currently blocked by this issue. Could also be a misunderstanding on my end...Would be great if you could assist.

**Comment by stevejroberts** [comment]
@daniel-lohausen-consilica It's been several years since I last looked at or worked with this code, so I'd probably be more of a hindrance than a help at this stage. I've alerted my former colleagues in the teams that now look after this toolkit to your message, hopefully we can get you some traction.

**Comment by rbbarad** [comment]
Thanks for bringing this issue to our attention! This has been fixed in #539 - resolving this issue
